### ISSUE TITLE
UI: Payment page expired and payed at the same time (voiding payment)

### ISSUE THREAD

**Issue by ndeet** [btcpayserver/btcpayserver#216]
Small UI hiccup which will rarely happen in real world but nevertheless looks a bit strange:
![image](https://user-images.githubusercontent.com/1136761/42402624-4251cbea-817c-11e8-824f-0ccdff996737.png)

When a user does a payment you can void the payment in BTCPay UI if it is not confirmed yet. Doing this while the user still has the payment page open without returning to the shop the displayed information is confusing as the invoice as paid + expired at the same time (see above).

How to reproduce:
- Pay on-chain, invoice marked as paid
- do NOT return to merchant shop
- in BTCPay backend UI make the order void manually before first confirmation

Result:
Paid + expired information is overlayed the same time.

Expected result:
Should switch to void state.

**Comment by NicolasDorier** [comment]
ping @rockstardev any idea?

**Comment by rockstardev** [comment]
Yeah, it has to do with that legacy `if/else` UI state transitions. Submitted the pull request that fixes it - @ndeet feel free to test on your server before merge, you just need to update `core.js`.

I've tested locally transitioning from paid -> expired -> paid -> invalid -> paid and UI updated correctly.
